pages/apis/rest_api/builds.md.erb

```diff
@@ -615,3 +615,70 @@ curl -X PUT "https://api.buildkite.com/v2/organizations/{org.slug}/pipelines/{pi
 Required scope: `write_builds`
 
 Success response: `200 OK`
+
+## Timestamp attributes
+
+There are several different timestamps relating to timing for builds and jobs. There are four main time values which are available on both build and job API calls.
+
+The timestamps are available via both the GraphQL and REST APIs. They differ slightly between the build and job objects.
+
+Each <em>build</em> is provided with the following timestamps:
+
+<table>
+<tbody>
+  <tr>
+    <th><code>scheduled_at</code></th>
+    <td>The time the build was created</td>
```

Comment:
For a `Build`  the value of `scheduled_at` is the time the Build is created, however for a `Job` the value of `scheduled_at` is actually the time that the scheduler process processes the job. It’s usually pretty much immediately following the job being created however, so often there is only a couple hundred ms in it.